File: Femis Assignments/Morning Snacks/StrongNumbers.js
const prompt = require("prompt-sync")();

let number = parseInt(prompt("Enter a number: "));
let actual = number;
let sum = 0;

while (actual > 0) {
let digit = actual % 10;


let fact = 1;
for (let i = 1; i <= digit; i++) {
    fact *= i;
}

sum += fact;
actual = parseInt(actual / 10);
}

if (sum === number) {
console.log(number + " is a Strong number");
} else {
console.log(number + " is NOT a Strong number");
}

File: Femis Assignments/Morning Snacks/StrongNumbers.py
number = int(input("Enter a number: "))
actual = number
sum_of_digits = 0

while actual > 0:
    digit = actual % 10

    fact = 1
    for i in range(1, digit + 1):
        fact *= i

    sum_of_digits += fact
    actual //= 10

if sum_of_digits == number:
    print(number, "is a Strong number")
else:
    print(number, "is NOT a Strong number")

